### PR TITLE
expo-feat: navigation header

### PR DESCRIPTION
**Pull request by Megio**
Created Navigation Header Component.
Styles could be changed if needed, the profile picture is hardcoded waiting for real data, and sign-out function has not been implemented yet.

Fixes: #1528 

Screenshot 2023-03-07 at 18 37 48

### CODE REVIEW

**Review by vyktoremario — DISMISSED** [review]

**Review by BigAB — CHANGES_REQUESTED** [review]
I would say this looks pretty good, but you need to run
```sh
yarn format
```
...and remove the `yarn-error.log` error from this PR.

You may want to look into configuring your editor to run `prettier` on save

File: expo-zustand-styled-components/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { GitHubLogo } from './GitHubLogo';
+import { StyledHeader } from './Header.styles';
+import UserDropdown from '../UserDropdown';
+
+const Header = () => {
+  return (
+    <StyledHeader>
+      {/* <Link to="/"> */}
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/components/Header/Header.tsx:9]: please remove comments as we aren't gonna use Link, yeah?

**Review by hdJerry — CHANGES_REQUESTED** [review]
Also, there are some conflicts that you need to look over. Thanks

**Review by BigAB — DISMISSED** [review]

**Review by AllStackDev1 — COMMENTED** [review]
This will help with the failing tests

https://react-native-async-storage.github.io/async-storage/docs/advanced/jest/

**Review by AllStackDev1 — DISMISSED** [review]

File: expo-zustand-styled-components/src/screens/Home/Home.tsx [hunk 2]
```diff
@@ -5,17 +5,18 @@ import { SafeAreaViewStyled } from './Home.styles';
...
+  const { token } = useAuthStore();
 
   return (
     <SafeAreaViewStyled>
+      <Header />
       <View>
         <Text>Hello World</Text>
         <Text>Here is the token{token}</Text>
-        <Button title="Logout" onPress={() => logout()} />
+        {/* <Button title="Logout" onPress={() => logout()} /> */}
```
Comment (hdJerry) [inline expo-zustand-styled-components/src/screens/Home/Home.tsx:19]: Please is there a need for this comment?

**Review by hdJerry — CHANGES_REQUESTED** [review]

**Review by hdJerry — APPROVED** [review]
Thanks Sir

**Review by AllStackDev1 — APPROVED** [review]

**Pull request merged**